cmake/sdks.cmake

```diff
@@ -4,6 +4,8 @@ set(SDK_DEPENDENCY_BUILD_LIST "")
 
 set(NON_GENERATED_CLIENT_LIST access-management text-to-speech core queues s3-encryption identity-management transfer)  ## Manually generated code with a name mimicking client name
 
+option(ENABLE_SMOKE_TESTS "Enable smoke tests" OFF)
```

Comment:
Please move this CMake option to the top level CMakeLists where the [other options are defined](https://github.com/aws/aws-sdk-cpp/blob/main/CMakeLists.txt#L31-L68)